src/Lumper.UI/ViewModels/MainWindowViewModel.IO.cs

```diff
@@ -32,23 +34,32 @@ private static IReadOnlyList<FilePickerFileType> GenerateBspFileFilter()
     {
         var bspFilter = new FilePickerFileType("BSP files")
         {
-            Patterns = new[] { "*.bsp" },
+            Patterns = new[]
+            {
+                "*.bsp"
+            },
```

Comment:
Right - the autoformatting commit will be at the end of the PR, meaning you'll just need to ignore cases of bad formatting. I know that makes the commits a bit uglier but I'd have to spend hours fixing merge conflicts to rebase those formatting changes into lower PRs.

Reply:
no, thats why I told you to watch out with the commits and why keeping formatting separate is important. We're not changing things back and forth in one PR. Might as well just not split anything and just do one big "refactor" commit if the supposedly clean version does useless changes and has wrong commit messages. Also for this changes it just worked, but depends on further changes in the file of course.

Reply:
Genuinely don't understand what you want me to do here. This was a comment on a single comment running the `dotnet` autoformatter, it is split up. If you want me to fixup the manual formatting changes I did later into this one I can, but regardless I don't know why you want to review the formatting changes anyway, at most you should just skim over the manual autoformatting one. As in my comment just now, we should discuss this on discord, ideally in voice. I don't really know how to make progress here otherwise.

Reply:
Going skip splitting this up since you've already gone through in review as we discussed yesterday, if still okay with you